### ISSUE TITLE
ModuleNotFoundError: No module named 'scrapy.settings.deprecated'

### ISSUE THREAD

**Issue by 474416133** [Gerapy/Gerapy#150]
**Describe the bug**
当使用命令 gerapy 时出现：
ModuleNotFoundError: No module named 'scrapy.settings.deprecated'

**To Reproduce**
1.  gerapy migrate便出现上面错误 

**Traceback**
gerapy migrate
Traceback (most recent call last):
  File "/Users/sven/opt/anaconda2/envs/spiders-manager/bin/gerapy", line 5, in <module>
    from gerapy.cmd import cmd
  File "/Users/sven/opt/anaconda2/envs/spiders-manager/lib/python3.7/site-packages/gerapy/cmd/__init__.py", line 5, in <module>
    from gerapy.cmd.parse import parse
  File "/Users/sven/opt/anaconda2/envs/spiders-manager/lib/python3.7/site-packages/gerapy/cmd/parse.py", line 2, in <module>
    from gerapy.server.core.parser import get_start_requests, get_follow_requests_and_items
  File "/Users/sven/opt/anaconda2/envs/spiders-manager/lib/python3.7/site-packages/gerapy/server/core/parser.py", line 6, in <module>
    from scrapy.settings.deprecated import check_deprecated_settings
ModuleNotFoundError: No module named 'scrapy.settings.deprecated'

**Environment (please complete the following information):**
 - OS: macOS Mojave
 - Python Version 3.7
 - Gerapy Version '0.9.2'
- Scrapy version (2, 1, 0)

**Additional context**

**Comment by AgentJiao** [comment]
问题解决了吗？遇到同样的问题。。

**Comment by AgentJiao** [comment]
下个旧版本scrapy（2.0.1版本），解压后将settings文件夹下的deprecated.py拷贝到当前scrapy环境下的settings中

**Comment by 474416133** [comment]
> 下个旧版本scrapy（2.0.1版本），解压后将settings文件夹下的deprecated.py拷贝到当前scrapy环境下的settings中

谢谢

**Comment by Cally99** [comment]
Is this fixed?

**Comment by Cally99** [comment]
Any comment? Project is looking like bug central

**Comment by borys25ol** [comment]
I installed an elder version of Scrapy and look likes it working correctly for now. I installed Scrapy==1.5

**Comment by Germey** [comment]
Bug fixed, 0.9.3 released.

Please try the new version of Gerapy:

```
pip3 install -U gerapy
```

or 

```
docker run -d -v ~/gerapy:/app/gerapy -p 8000:8000 germey/gerapy:0.9.3
```

Thanks for your support.
